client/src/pages/schema/IndexTypeElement.tsx

```diff
@@ -86,18 +92,15 @@ const IndexTypeElement: FC<{
   const DeleteIcon = icons.delete;
 
   const fetchStatus = useCallback(async () => {
-    if (data._indexType !== '') {
+    // prevent delete index trigger fetching index status
+    if (data._indexType !== '' && status !== 'delete') {
```

Comment:
Can we have enum for empty status and delete status, eg: STATUS.EMPTY , STATUS.DELETE

Reply:
Milvus has index status enum, but `empty` and `delete` status only needed for ui. Do you think it's better to add them?

<img width="391" alt="indexStatus" src="https://user-images.githubusercontent.com/20420181/128801340-2c5596d0-db20-4c46-8457-bf6777e53e75.png">

Reply:
Yeah, I think so.  You can leave a comment on these status, they are UI only.